### ISSUE TITLE
Guzzle

### ISSUE THREAD

**Issue by emresaracoglu** [gnello/php-mattermost-driver#45]
Hello,

It was working great in my old project but I noticed a version issue in my new project. Is there any way to fix it?

Thank you.

```
Using version ^2.16 for gnello/php-mattermost-driver
./composer.json has been updated
Running composer update gnello/php-mattermost-driver
Loading composer repositories with package information
Updating dependencies
Your requirements could not be resolved to an installable set of packages.

  Problem 1
    - Root composer.json requires gnello/php-mattermost-driver ^2.16 -> satisfiable by gnello/php-mattermost-driver[v2.16.0].
    - gnello/php-mattermost-driver v2.16.0 requires guzzlehttp/guzzle ^6.2 -> found guzzlehttp/guzzle[6.2.0, ..., 6.5.5] but it conflicts with your root composer.json require (^7.4).

Use the option --with-all-dependencies (-W) to allow upgrades, downgrades and removals for packages currently locked to specific versions.
You can also try re-running composer require with an explicit version constraint, e.g. "composer require gnello/php-mattermost-driver:*" to figure out if any version is installable, or "composer require gnello/php-mattermost-driver:^2.1" if you know which you need.

Installation failed, reverting ./composer.json and ./composer.lock to their original content.
```

**Comment by gnello** [comment]
Hi @emresaracoglu,
fixed within [v2.16.1](https://github.com/gnello/php-mattermost-driver/tree/v2.16.1)!

Thank you for bringing that to my attention :)

**Comment by emresaracoglu** [comment]
I thank you for this repo. @gnello
